### ISSUE TITLE
Honor `tagOwners` when tagging devices

### ISSUE THREAD

**Issue by vbrandl** [iFargle/headscale-webui#87]
I'm using Authelia as OIDC provider to authenticate in headscale-webui.
In my headscale ACL, I configured a non-existent user as the tagOwner for `testtag`:

```json
"tagOwners": {
  "tag:testtag": ["some-nonexisting-user"]
}
```

Then I logged into headscale-webui and tagged a device with `testtag`. I would expect this to fail but the tag was created.

**Comment by iFargle** [comment]
This seems more of an issue with Headscale than headscale-webui, no?  
This UI does nothing with the ACL's in Headscale yet.

**Comment by vbrandl** [comment]
Maybe I misunderstood the OIDC auth. I thought it is used to make only user specific settings available. But since an API key is used to call the headscale API, the permissions of that API key are used for everyone

**Comment by iFargle** [comment]
Yep, it's pretty simplistic :)  OIDC auth is only for auth to headscale-webui itself -- Completely detached from headscale currently.
